todo_app/app.py

```diff
@@ -3,40 +3,284 @@
 
 
 from todo_app.flask_config import Config
-from todo_app.data.session_items import get_items
-from todo_app.data.session_items import add_item
-from todo_app.data.session_items import get_item
-from todo_app.data.session_items import save_item
+from todo_app.data.session_items import get_items, add_item, get_item, save_item
+import requests
+import json
+import os
+import logging
+
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
 
+#@app.route('/')
+
+#r= requests.get("https://api.trello.com/1/members/me/boards?fields=name,url&key=77f5ab920b78a542ed2fb7501190d22d&token=1b81c916acc9144aa57afc7decdd1249d033e0bf52d300c5587dd38aa3acc846")
+#r.content
+
+#r = "https://api.trello.com/1/boards/6005828032dafa5707bf5dc3"
+#url = "https://api.trello.com/1/boards/{id}"
+#url = "https://api.trello.com/1/members/me/boards?fields=name,url"
+#&key=77f5ab920b78a542ed2fb7501190d22d&token=1b81c916acc9144aa57afc7decdd1249d033e0bf52d300c5587dd38aa3acc846"
+#id = "6005828032dafa5707bf5dc3"
+#url = "https://api.trello.com/1/boards/6005828032dafa5707bf5dc3/lists" #get board lists
+#url = "https://api.trello.com/1/boards/6005828032dafa5707bf5dc3/cards/6012a4a01353064030144e40"
+#url = "https://api.trello.com/1/boards/6005828032dafa5707bf5dc3/cards"
+#url = "https://api.trello.com/1/boards/6005828032dafa5707bf5dc3/labels"
+#url = "https://api.trello.com/1/cards/6005828032dafa5707bf5de4/list"
+#url = "https://api.trello.com/1/members/me/board"
+#url = "https://api.trello.com/1/boards/6005828032dafa5707bf5dc3/cards"
+#url="https://api.trello.com/1/boards/6005828032dafa5707bf5dc3/cards"
+
+def getToDo():
+	url = "https://api.trello.com/1/lists/6005828032dafa5707bf5dc5/cards"
```

Comment:
All of these hardcoded list ids mean that your application can only be used with your particular trello board. Instead, we should either:
* Grab these dynamically (e.g. checking /boards/{board_id}/lists
* Grab these from environment variables

Reply:
Thank you for the suggestion, I have updated it with commit c58b512, 
bc48505 and 
22f3884